.scaffolder/plugin-feature/feature.php.hbs

```diff
@@ -0,0 +1,27 @@
+<?php
+/**
+ * {{ wpClassName inputs.featureName }} class file
+ *
+ * @package Create_WordPress_Plugin
```

Comment:
Need to ensure find and replace is run, this should be (imo) Alley_SEO

Reply:
WP_SEO is fine for now, we can make a call later if we want to rename it to avoid confusion, especially with Yoast (which is `wordpress-seo`)